training/tests/integration/conftest.py

```diff
@@ -194,7 +194,66 @@ def gnn_config(
 
     cfg = OmegaConf.merge(template, testing_modifications_with_temp_dir, use_case_modifications)
     OmegaConf.resolve(cfg)
-    return cfg, dataset_urls[0]
+    return cfg
+
+
+#TODO make graphtransformer_1g the default test case, and only run that
+#TODO add small o48 test case which can be run anywhere
+#TODO change it so i can inherit from stretched_config etc. rather then rewriting
+@pytest.fixture(
+    params=[ #selects different test cases
+        #"graphtransformer_n320_1g",
+        #"gnn_n320_1g",
+        #"stretched",
+        "lam",
+    ],
+)
+def benchmark_config(
```

Comment:
I think it could be better to have a:
- benchmark_config for the global
- benchmark_lam_config
- benchmark_stretched_config
so that way we have a similar set up as in the 'normal integration tests' and we keep the functions clearer